app/controllers/announcements_controller.rb

```diff
@@ -17,6 +17,19 @@ def create
     end
   end
 
+  def destroy
+    @announcement = coronavirus_page.announcements.find(params[:id])
+    @coronavirus_page = @announcement.coronavirus_page
```

Comment:
Minor: This line isn't needed, as we already know which `coronavirus_page` we're on.